integration-tests/test_gcs.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+import io
+import os
+
+import google.cloud.storage
+
+import smart_open
+
+_GCS_BUCKET = os.environ.get('SO_GCS_BUCKET')
+_GCS_URL = 'gs://' + _GCS_BUCKET
```

Comment:
Why `gs://` when the service is GCS?

Reply:
`gs://` is the most common scheme for Google Cloud Storage, but `gcs://` is also used occasionally

Reply:
Is there some specification for this? Where does this convention come from, where is it used?

Reply:
[Google's tools](https://cloud.google.com/bigquery/docs/loading-data-cloud-storage#gcs-uri) (gsutil, Google Cloud Storage console uris) all use `gs`, I actually had not seen `gcs` before the issue that requested this feature, and think maybe we should drop `gcs` for simplicity.

Reply:
OK, Google itself = the originator of the protocol is a source that's authoritative enough :)

Reply:
I agree, it's very confusing because they use the GCS acronym to refer to the Google Cloud Storage service. I wouldn't be surprised is their reasoning was one less letter to send in network calls, sounds about right for Google haha.

To clarify, where did we land with this? Just support `gs` or both `gs` and `gcs`?

Reply:
I'm +1 on `gs` only, if that's the established standard. @mpenkov thoughts?

Reply:
Yes, `gs` is fine initially. If people complain, we can revisit later.